web/source/keymanweb.js

```diff
@@ -3681,21 +3722,30 @@
     } else {
       if(Pelem.tagName.toLowerCase() != 'iframe') {
         if(keymanweb.isKMWInput(Pelem)) {
-          var x = keymanweb.setupTouchElement(Pelem);
-
-          if(x) {
-            // if needed, let the timeout variable be: keymanweb.touchInputAddedTimer
-            window.setTimeout(function() {
-              keymanweb.updateInput(x);
-            }, 1);
-          }
+          keymanweb.setupTouchElement(Pelem);
```

Comment:
It'll also help us to do further optimizations later, as we can skip certain things if we know we're handling a non-touch element.